MASFoundation/Classes/models/MASConfiguration.m

```diff
@@ -889,6 +891,22 @@ - (BOOL)enabledTrustedPublicPKI
     return [mobileInfo[MASMobileEnbaledTrustedPublicPKI] boolValue];
 }
 
+
+- (BOOL)sslpinningEnabled
```

Comment:
can you please double-check the naming convention here?
The right one could be: ssl**P**inningEnabled